src/main/java/pl/javastart/task/Drink.java

```diff
@@ -0,0 +1,10 @@
+package pl.javastart.task;
+
+public class Drink {
+    String drinkName;
```

Comment:
jeżeli jesteśmy w klasie Drink, to nie ma potrzeby dodawania "drink" do nazw pól. Wiadomo, że nazwa dotyczy drinka i wystarczy samo "name"

Reply:
Poprawione.